hack/get_ci_vm.sh

```diff
@@ -69,9 +69,10 @@ delvm() {
 }
 
 image_hints() {
+    _BIS=$(egrep -m 1 '_BUILT_IMAGE_SUFFIX:[[:space:]+"[[:print:]]+"' "$BUILDAHROOT/.cirrus.yml" | cut -d: -f 2 | tr -d '"[:blank:]')
```

Comment:
I think I'd feel happier with a `test -n "${_BIS}" || die`

Reply:
k, got it.